compare-bug-version-and-git-branch.sh

```diff
@@ -23,6 +23,9 @@ BUG=$(git show --name-only --format=email | awk '{IGNORECASE=1} /^BUG:/{print $2
 if [ -z "${BUG}" -a "${GERRIT_TOPIC}" = "rfc" ]; then
     echo "No BUG id for rfc needed."
```

Comment:
Should we instead add a check here? Patches with rfc should have an issue?

@ShyamsundarR Thoughts?